service/src/main/java/bio/terra/workspace/db/WorkspaceActivityLogDao.java

```diff
@@ -0,0 +1,59 @@
+package bio.terra.workspace.db;
+
+import bio.terra.common.db.ReadTransaction;
+import bio.terra.common.db.WriteTransaction;
+import bio.terra.workspace.db.model.DbWorkspaceActivityLog;
+import java.time.Instant;
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.Optional;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.jdbc.core.namedparam.MapSqlParameterSource;
+import org.springframework.jdbc.core.namedparam.NamedParameterJdbcTemplate;
+import org.springframework.stereotype.Component;
+
+@Component
+public class WorkspaceActivityLogDao {
+
+  private final NamedParameterJdbcTemplate jdbcTemplate;
+
+  @Autowired
+  public WorkspaceActivityLogDao(NamedParameterJdbcTemplate jdbcTemplate) {
+    this.jdbcTemplate = jdbcTemplate;
+  }
+
+  @WriteTransaction
+  public void writeActivity(UUID workspaceId, DbWorkspaceActivityLog dbWorkspaceActivityLog) {
+    switch (dbWorkspaceActivityLog.getOperationType()) {
+      case UNKNOWN:
```

Comment:
I'm confused. You went to the trouble of passing these in, but you are not using them?